src/surrealdb/mod.rs

```diff
@@ -134,7 +134,7 @@ mod tests {
         opt::auth::Root,
         Surreal,
     };
-    use testcontainers::{runners::AsyncRunner, ImageExt};
+    use testcontainers::runners::AsyncRunner;
```

Comment:
Issue in tests wasn't related to surreldb import, I'd prefer not to touch unrelated modules here
It was a warning, but not causing the failure of test suite. See comment above